bridge/qb/server/events.lua

```diff
@@ -38,4 +41,22 @@ RegisterServerEvent('baseevents:leftVehicle', function(veh, seat, modelName, net
         event = 'Left'
     }
     TriggerClientEvent('QBCore:Client:VehicleInfo', src, data)
+end)
+
+---Compatability event for logging
+---@param name string source of the log. Usually a playerId or name of a script.
+---@param title string the action or 'event' being logged. Usually a verb describing what the name is doing. Example: SpawnVehicle
+---@param message string the message attached to the log
+---@param color? string what color the message should be
+---@param tagEveryone? boolean Whether a role tag should be applied to this log. Uses config variable for logging role
+---@deprecated use logger module from qbx_core
+RegisterNetEvent('qb-log:server:CreateLog', function(name, title, color, message, tagEveryone)
+    logger.log({
+        source = GetInvokingResource() or "qbx_core",
+        webhook = name,
+        event = title,
+        message = message,
+        color = color,
+        tags = tagEveryone and config.logging.role or nil
+    })
```

Comment:
I've discussed why I'm against having this event in the first place as core doesn't own this event; small resources does. If you do want to keep it, it needs to be disabled by default.

Reply:
I'll just remove it. Without having a centralized location for the webhook matching it will just confuse folks more than it'll help. I'll just publish a gist that can be linked for someone to drop in if they want it.